plugins/pom-analyzer/pom.xml

```diff
@@ -10,15 +9,64 @@
     <artifactId>pom-analyzer</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>dev.c0ps</groupId>
+            <artifactId>commons-maven</artifactId>
+            <version>0.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>dev.c0ps.maven-easy-index</groupId>
+            <artifactId>data</artifactId>
+            <version>0.0.1</version>
+        </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-model</artifactId>
-            <version>3.8.4</version>
+            <version>3.9.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-artifact</artifactId>
+            <version>3.9.3</version>
         </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-model-builder</artifactId>
-            <version>3.8.4</version>
+            <version>3.9.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap.resolver</groupId>
+            <artifactId>shrinkwrap-resolver-depchain</artifactId>
+            <version>3.2.1</version>
+            <type>pom</type>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap.resolver</groupId>
+            <artifactId>shrinkwrap-resolver-impl-maven</artifactId>
+            <version>3.2.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>4.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.16</version>
+        </dependency>
+
+        <!-- metadata db -->
+        <dependency>
+            <groupId>eu.fasten</groupId>
+            <artifactId>core-metadata</artifactId>
+            <version>0.0.19</version>
+        </dependency>
+        <dependency>
+            <!-- TODO remove this dependency (functionality should be native since jOOQ v3.15 -->
```

Comment:
Could you remove this unnecessary dependency in this PR as mentioned in the comment?

Reply:
I tried it, but it did not work without it... I am not an expert on jooq, so I left it in place for now.